test/Typewriter.Test/TestDataPHP/com/Microsoft/Graph/CallRecords/Model/CallRecord.php

```diff
@@ -59,7 +59,7 @@ public function setVersion($val)
     public function getType()
     {
         if (array_key_exists("type", $this->_propDict)) {
-            if (is_a($this->_propDict["type"], "Microsoft\Graph\CallRecords\Model\CallType")) {
+            if (is_a($this->_propDict["type"], "\Microsoft\Graph\CallRecords\Model\CallType")) {
```

Comment:
@MIchaelMainer just curious...

For `v1` models, why do we generate the CallRecords models in their own namespace outside the `\Microsoft\Graph\Model` namespace? 

Also noticed that for `beta` we have `Ediscovery` and `TermStore` outside `Beta\Microsoft\Graph\Model` namespace as well.

Reply:
I shared the namespaces design document with you and Kenneth.